docs/notes.md

```diff
@@ -878,5 +881,438 @@ With these observations, we can simplify the combined \\(m\\)-party statement ab
 \end{aligned}
 \\]
 
+Arithmetic Circuit Proofs
+=========================
+
+The goal of an *arithmetic circuit proof* is for a prover to convince a verifier that a particular set of values \\(v\\) satisfy the constraints represented by the arithmetic circuit, without revealing any additional information about the values \\(v\\).
```

Comment:
I think it might be good here to briefly say what an arithmetic circuit is, before saying what we're trying to prove about one? A few paragraphs down there's mention of "one type of gate", but there's not a prior definition of a circuit as a DAG of addition and multiplication gates.